src/player.cpp

```diff
@@ -7456,6 +7456,58 @@ void player::rooted()
     }
 }
 
+bool player::add_faction_warning( const faction_id id )
```

Comment:
```suggestion
bool player::add_faction_warning( const faction_id &id )
```
And change the header accordingly.